sentry_sdk/tracing.py

```diff
@@ -1400,12 +1400,18 @@ def name(self, value):
     @property
     def source(self):
         # type: () -> str
-        pass
+        from sentry_sdk.integrations.opentelemetry.consts import SentrySpanAttribute
+
+        return (
+            self._get_attribute(SentrySpanAttribute.SOURCE) or TRANSACTION_SOURCE_CUSTOM
```

Comment:
`get_attribute` is now public on current potel-base as of https://github.com/getsentry/sentry-python/commit/a67125ae6cd25f3674e0d7ad950a1e0662ac302f
```suggestion
            self.get_attribute(SentrySpanAttribute.SOURCE) or TRANSACTION_SOURCE_CUSTOM
```